### DISCUSSION TITLE
Deterministic position of human body

### DISCUSSION THREAD

**Discussion by martinschonger** [empriselab/RCareWorld#137] [Help - Programming] [answered]
For development purposes, it would be helpful to somehow deterministically control the position of the human body, e.g. either directly or by setting some random seed. I'm aware of #127, but still, maybe a dedicated 'dev executable' would be an option? It was already pointed out to me that the desired functionality exists (cf. [here](https://github.com/empriselab/RCareWorld/blob/phy-robo-care/pyrcareworld/pyrcareworld/demo/examples/example_humanbody_ik.py)), but is disabled in the current executable (for good reason!).

Also linking https://github.com/empriselab/RCareWorld/discussions/125#discussioncomment-10767482, since there the idea of a dedicated executable for training/dev is mentioned as well.

**Comment by thisjustin123** [comment]
I'll add the ability to fix the randomization as a documented issue / feature request (#139). I'm focusing on joint velocity right now but will try to get to it soon!

**Reply by thisjustin123** [reply, marked as answer]
#141
